aptos-move/framework/src/natives/mod.rs

```diff
@@ -113,5 +114,10 @@ pub fn all_natives(
         );
     }
 
+    add_natives_from_module!(
```

Comment:
I don't see anything here that indicates `confidential_proof` lies in `0x7` or `aptos_experimental`...

Even so, will this work on `testnet` and `mainnet`? Since `0x7` is experimental and not deployed there.